syntax = "proto3";

package moose;

service ChannelManager {
    rpc GetValue(GetValueRequest) returns(GetValueResponse) {}
}

message GetValueRequest {
    string rendezvous_key = 1;
    int64 session_id = 2;
}

message GetValueResponse {
    bytes value = 1;
}
